backend/routers/controllers/user_controllers/UserService.js

```diff
@@ -0,0 +1,28 @@
+const bcrypt = require('bcrypt');
+const sequelize = require('sequelize');
+const userModel = require('../../../db/models/user_models/UserModel.js');
+
+const sequelizeOperators = sequelize.Op;
+
+
+const salt = bcrypt.genSaltSync(10);
+
+function createUserAccount(registrationRequest, response) {
```

Comment:
You are in the user service, so I think it's safe to change it to createAccount or just create, it's obvious that userService isn't going to create treeAccounts